content/sensu-core/1.7/reference/clients.md

```diff
@@ -108,6 +108,12 @@ _WARNING: registration events are not stored in the event registry (in the Sensu
 [data store][8]), so they are not accessible via the Sensu API; however, all
 registration events are logged in the [Sensu server][5] log._
 
+#### Deregistration events
+
+Similarly to registration events, Sensu can create and process a deregistration event when the Sensu client process stops.
+You can use deregistration events to trigger a handler that updates external CMDBs or performs an action to update ephemeral infrastructures.
+To enable deregistration events, set the `deregister` flag to `true` and specify the event handler using the [`deregistration` definition scope][48].
```

Comment:
This line probably needs words to clarify that these settings are configured on the client side.